Documentation/commands.md

```diff
@@ -317,6 +316,46 @@ Work in progress. Please contribute!
 
 Work in progress. Please contribute!
 
+### Logging
+
+By default, rkt will send logs directly to stdout/stderr, allowing them to be caputered by the invoking process.
+On host systems running systemd, rkt will attempt to integrate with journald on the host.
+In this case, the logs can be accessed directly via journalctl.
+
+NOTE: Journald integration requires systemd version v219 or v220 in stage1.
```

Comment:
I would generally shy away from such statements, stage1 should be an implementation detail that most users don't need to care about. Not a huge problem here since this is temporary, but in future it would be better to try talk about it a bit differently